src/components/CurrentWalletBalance.js

```diff
@@ -41,7 +47,9 @@ const CurrentWalletBalance = (props) => {
     }
 
     const formattedBalance = props.numberFormat(
-        props.userWallet.currentBalance / 100, // Divide by 100 because balance is in cents
+
+        // Divide by 100 because balance is in cents. Default to zero if userWallet and currentBalance is not set yet to avoid NaN
+        (props.userWallet.currentBalance || 0) / 100,
```

Comment:
Instead of doing putting a default of `0` here we can put it in the default props.

Reply:
Issue is this originally showed a spinner when userWallet onxy key is empty. But now we want to show $0 when offline.